<?xml version="1.0"?>
<launch>
 <node pkg ="rosbag" type="play" name="rosbag" args="-l $(find mpu6050_new)/bag/mpu.bag"/>
 <node name="moving_avg" pkg="mpu6050_new" type="moving_avg.py" />
 <node name="serial_comm" pkg="mpu6050_new" type="serial_comm.py" />

 </launch>
